### PR TITLE
Adding README to ocm script

### PR DESCRIPTION
**Pull request by mrrajan**
Added a README.md file

### CODE REVIEW

File: utils/scripts/ocm/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Provision Openshift Dedicated Cluster in AWS
```
Comment (aloganat) [inline utils/scripts/ocm/README.md:1]: Add "Documentation" label to your pull request

Comment (aloganat) [inline utils/scripts/ocm/README.md:1]: The script also does other things more than provisioning OSD cluster. So, Can we say like , Openshift Dedicated Cluster Operations in AWS ?

File: utils/scripts/ocm/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Script to install Openshift Dediated (OSD) on AWS with the aid of Openshift Cluster Manager CLI tool ([ocm-cli](https://github.com/openshift-online/ocm-cli.git))
+
+# Prerequisites
+- Python 3.x
+- Install ocm cli
```
Comment (aloganat) [inline utils/scripts/ocm/README.md:6]: Give a for example link to one of the latest ocm cli version to install

File: utils/scripts/ocm/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    
+### Login to OCM
+Before creating a cluster, user should login to OCM using access tokens.
+Retrieve the access token from [here](https://console.redhat.com/openshift/token/show) and login to ocm by running the below script
+    
+``` python ocm.py ocm_login --token "<token_here>"  --url=staging```
+    
+Replace the field <token_here>.
+
+*Note:* Argument --url=staging sets the targeted environment for OSD to deploy. _url=staging_ deploys a cluster in [staging](https://qaprodauth.cloud.redhat.com/) environment. If the argument removed, cluster would get deployed in [production](https://console.redhat.com/) environment
```
Comment (aloganat) [inline utils/scripts/ocm/README.md:79]: Please remove the stage environment link.

File: utils/scripts/ocm/README.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+*Note:* AWS account access_key and secret_accesskey for the user ocdCcsAdmin should be used to create a cluster, please refer [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) to get more details about AWS identity and Access Management.
+
+### Install RHODS addon
+    
+```python3 ocm.py install_rhods_addon --cluster-name <clustername>```
+    
+Replace the field <clustername> to the cluster name deployed
+
+### Create IDP
```
Comment (aloganat) [inline utils/scripts/ocm/README.md:96]: The script currently supports creating htpasswd and ldap IDP. Could you also give an example for ldap IDP

File: utils/scripts/ocm/README.md (hunk 2 above)
Comment (rravi6121) [reply utils/scripts/ocm/README.md]: Added a hyperlink to the section "Install OCM-CLI" which has example to install latest ocm cli version

**Review by aloganat — DISMISSED** [review]
LGTM

File: utils/scripts/ocm/README.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
+# Openshift Dedicated Cluster operations in AWS with OCM CLI and Openshift API
+Script to provision Openshift Dediated (OSD) on AWS and with other functionalities like creating identity providers(IDP) to access the cluster, installing/uninstalling add-ons and retreiving the cluster info with the aid of Openshift Cluster Manager CLI tool ([ocm-cli](https://github.com/openshift-online/ocm-cli.git))
```
Comment (lugi0) [inline utils/scripts/ocm/README.md:2]: Typo in `Dediated`

Comment (lugi0) [reply utils/scripts/ocm/README.md:2]: typo in `retreiving`

File: utils/scripts/ocm/README.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+# Prerequisites
+- Python 3.x
+- [Install ocm cli](#Install%20OCM-CLI)
+- access_key and secret_access_key for the AWS account user *ocdCcsAdmin*
```
Comment (lugi0) [inline utils/scripts/ocm/README.md:7]: What account is `ocdCcsAdmin`?

File: utils/scripts/ocm/README.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+
+The above command creates IDP type of htpasswd with the default IDP name htpasswd-cluster-admin and User name htpasswd-cluster-admin-user
+   
+##### LDAP
+
+```python3 ocm.py create_idp --type ldap --cluster <cluster_name> --ldap-bind-password <ht_password>```
+    
+Replace the fields <cluster_name> and <ht_password>
+    
+The above command creates IDP type of ldap with the password given
```
Comment (lugi0) [inline utils/scripts/ocm/README.md:119]: Formatting is broken, everything is treated as a code block

File: utils/scripts/ocm/README.md (hunk 6 above)
Comment (rravi6121) [reply utils/scripts/ocm/README.md:7]: Hi @lugi0, This is a aws user who has policy (_**OSDRequirement**_) defined to deploy OSD on AWS

File: utils/scripts/ocm/README.md (hunk 5 above)
Comment (rravi6121) [reply utils/scripts/ocm/README.md]: Ah, missed it out.

File: utils/scripts/ocm/README.md (hunk 7 above)
Comment (rravi6121) [reply utils/scripts/ocm/README.md]: Not sure where it is broken, but the web view loads as expected.

File: utils/scripts/ocm/README.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
+# Openshift Dedicated Cluster operations in AWS with OCM CLI and Openshift API
+Script to provision Openshift Dedicated (OSD) on AWS and with other functionalities like creating identity providers(IDP) to access the cluster, installing/uninstalling add-ons and retrieving the cluster info with the aid of Openshift Cluster Manager CLI tool ([ocm-cli](https://github.com/openshift-online/ocm-cli.git))
```
Comment (tarukumar) [inline utils/scripts/ocm/README.md:2]: "Cluster" should be included after Openshift Dedicated (OSD).

Comment (rravi6121) [reply utils/scripts/ocm/README.md:2]: @tarukumar on the title?

Comment (tarukumar) [reply utils/scripts/ocm/README.md:2]: Nope in the second line .:

**Script to provision Openshift Dedicated (OSD) cluster on AWS**

**Review by jgarciao — COMMENTED** [review]
lgtm

File: utils/scripts/ocm/README.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
+# Openshift Dedicated Cluster operations in AWS with OCM CLI and Openshift API
+Script to provision Openshift Dedicated (OSD) cluster on AWS and with other functionalities like creating identity providers(IDP) to access the cluster, installing/uninstalling add-ons and retrieving the cluster info with the aid of Openshift Cluster Manager CLI tool ([ocm-cli](https://github.com/openshift-online/ocm-cli.git))
+
+# Prerequisites
+- Python 3.x
+- [Install ocm cli](#Install%20OCM-CLI)
+- access_key and secret_access_key for the AWS account user *ocdCcsAdmin*
```
Comment (tarukumar) [inline utils/scripts/ocm/README.md:7]: could you change the above line to something like this:

  access_key and secret_access_key for the "osdCcsAdmin" AWS Identity and Access Management (IAM) user with the AdministratorAccess

**Review by aloganat — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]

**Pull request merged**